pkg/process/procutil/process_model.go

```diff
@@ -23,7 +25,8 @@ type Process struct {
 	Uids     []int32
 	Gids     []int32
 
-	Stats *Stats
+	Stats    *Stats
+	Language *languagemodels.Language
```

Comment:
It seems that we no longer need to update `procutil.Process` with Language, right?

Reply:
That's correct, let me revert this. I will still keep the change in `languagedetection` however, since something like this will likely be added back when we add `Language` to `CollectorProc`